app/javascript/src/components/Dashboard/Notes/Pane/EditNote.jsx

```diff
@@ -1,16 +1,17 @@
 import React from "react";
 
-import { Pane, Typography } from "neetoui";
+import { Pane, Typography, Toastr } from "neetoui";
+
+import formInitialValues from "constants/formInitialValues";
 
 import Form from "./Form";
 
-export default function EditNotePane({
-  fetchNotes,
-  showPane,
-  setShowPane,
-  note,
-}) {
+const EditNotePane = ({ showPane, setShowPane }) => {
```

Comment:
Why have you suffixed with keyword `Pane` when this is already within Pane folder?